coverage/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2021-2024 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+    http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.google.fhir.gateway</groupId>
+    <artifactId>fhir-gateway</artifactId>
+    <version>0.3.2</version>
+  </parent>
+
+  <groupId>com.google.fhir.gateway</groupId>
+  <artifactId>coverage</artifactId>
+  <name>coverage</name>
+  <description>Compute aggregated test code coverage</description>
+  <packaging>pom</packaging>
+
+  <properties>
+    <maven.deploy.skip>true</maven.deploy.skip>
+    <root.basedir>${project.parent.basedir}</root.basedir>
+  </properties>
+
+  <dependencies>
```

Comment:
I see that the `exec` module contains few examples on how to use the `fhir-gateway` application. Do we need to include this module as well? From a coverage point it will be good as we will know if the custom changes made are good enough. WDYT?

Reply:
That module has no meaningful unit-test that covers `plugin` or `server` modules, hence including it in the report will not make a difference on lines that are covered. Also because that code is mostly example code, I don't think we need to add unit-tests for it.